CMakeLists.txt

```diff
@@ -186,6 +188,8 @@ add_executable(${TEST_EXE_NAME}
 		test/test_config.cpp
 		test/test_logger.cpp
 		test/test_main.cpp
+		src/app/log.cpp
+		src/app/clock.cpp
```

Comment:
This should also be included in `add_executable(${EXE_NAME} ...)`, shouldn't it?

Reply:
And it is